src/Pages/Signup/Signup.jsx

```diff
@@ -0,0 +1,215 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import UserInput from '../../Components/UserInput';
+import UserButton from '../../Components/UserButton';
+import './Signup.scss';
+
+const Signup = () => {
+  // 이미지 업로드
+  const [imageName, setImageName] = useState('');
+  // 유저 정보(필수 사항)
+  const [userInfo, setUserInfo] = useState({
+    email: '',
+    password: '',
+    passwordConfir: '',
+    nickname: '',
+  });
+  // 이메일
+  const userEmail = event => {
+    setUserInfo({ ...userInfo, email: event.target.value });
+  };
+  // 비밀번호
+  const userPassword = event => {
+    setUserInfo({ ...userInfo, password: event.target.value });
+  };
+  // 비밀번호 확인
+  const userPasswordConfir = event => {
+    setUserInfo({ ...userInfo, passwordConfir: event.target.value });
+  };
+  // 닉네임
+  const userNickName = event => {
+    setUserInfo({ ...userInfo, nickname: event.target.value });
+  };
+
+  // 이메일, 비밀번호, 비밀번호 확인 유효성 검사
+  const isVaild =
+    userInfo.email.includes('@') &&
+    userInfo.email.includes('.') &&
+    userInfo.password.length >= 10 &&
+    userInfo.password === userInfo.passwordConfir;
+
+  // 이미지 업로드 input
+  const handleFileChange = event => {
+    const file = event.target.files[0];
+    if (file) {
+      setImageName(file.name);
+    }
+  };
+  // 로그인 페이지 이동
+  const moveNavigate = useNavigate();
+  const goToLogin = () => {
+    moveNavigate('/');
+  };
+  // 회원가입 완료 페이지 이동
+  const goToComplete = () => {
+    fetch('/data/Signup.json', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8',
+      },
+      body: JSON.stringify({
+        email: userInfo.email,
+        password: userInfo.password,
+        nickname: userInfo.nickname,
+      }),
+    })
+      .then(response => {
+        if (response.ok) {
+          return response.json();
+        }
+        throw new Error('서버 응답이 올바르지 않습니다.');
+      })
+      .then(data => {
+        if (data.message === 'SIGNUP SUCCESS') {
+          moveNavigate('/signupcomplete');
+        } else {
+          alert('회원가입에 실패했습니다. 다시 시도해주세요.');
+        }
+      })
+      .catch(error => {
+        alert('회원가입 중 오류가 발생했습니다. 다시 시도해주세요.');
+      });
+  };
+
+  return (
+    <div className="signup">
+      <div className="signupInputFrame">
+        <div className="backButtonFrame">
+          <img
+            className="backIcon"
+            src="/../images/Back_arrow.svg"
```

Comment:
이 부분을 말한 거였습니다.

```suggestion
            src="/images/Back_arrow.svg"
```

기억나시죠?

Reply:
아하 넵! 기억났습니다! 바로 수정해서 적용시킬께요!

Reply:
완료했습니다.